packages/signature-controller/src/SignatureController.test.ts

```diff
@@ -533,7 +531,6 @@ describe('SignatureController', () => {
             { parseJsonData: true },
           ),
       );
-      expect(error instanceof EthereumProviderError).toBe(true);
       expect(error.message).toBe('User rejected the request.');
```

Comment:
For extra stability, is there value in including a `data` property in the thrown error also, to ensure that too is thrown unaltered?

Reply:
done